documentation/src/main/asciidoc/topics/dependencies/wfly_endpoints.adoc

```diff
@@ -0,0 +1,11 @@
+[source,xml,options="nowrap"]
+----
+<subsystem xmlns="urn:infinispan:server:endpoint:9.2">
```

Comment:
question, is it normal we do still use 9.2 and not 9.4 here?

Reply:
I think this is just one that hasn't been updated. I've fixed it to be 9.4. we'll have to update in 10 again.